client/lib/media/test/actions.js

```diff
@@ -69,6 +69,7 @@ describe( 'MediaActions', function() {
 						return {
 							get: mediaGet.bind( [ siteId, mediaId ].join() ),
 							update: mediaUpdate.bind( [ siteId, mediaId ].join() ),
+							edit: mediaUpdate.bind( [ siteId, mediaId ].join() ),
```

Comment:
Okay, I'm still not sure where `MediaActions.edit` is used. Nevermind, it's out of scope of this PR anyways. :)